src/Symfony/Component/Form/FormFactory.php

```diff
@@ -67,8 +67,8 @@ public function createNamedBuilder($name, $type = 'Symfony\Component\Form\Extens
             $options['data'] = $data;
         }
 
-        if (!\is_string($type)) {
-            throw new UnexpectedTypeException($type, 'string');
+        if (!\is_string($type) && !\is_int($type)) {
+            throw new UnexpectedTypeException($type, 'string|int');
```

Comment:
Is this correct? The form type can be an integer? the other changes are referring to the form name though.

Reply:
I agree we cannot do this change due to explicit type hint in FormBuilder's constructor (https://github.com/symfony/symfony/blob/a3b45eb03407773ee9e19e00cf93b98c22d28a65/src/Symfony/Component/Form/FormBuilder.php#L40). I would actually force strings for simplicity.

Reply:
@vudaltsov, @xabbuh I'm **not** referring to the form "name", but the "type", which cannot be an integer.

Reply:
🙈 I totally missed your point, fixed now

Reply:
@xabbuh , there's still `|int` in exception which has to be reverted if I get it right :)